docs/resources/Voice.md

```diff
@@ -11,7 +11,7 @@ Used to represent a user's voice connection status.
 | guild_id? | snowflake | the guild id this voice state is for |
 | channel_id | ?snowflake | the channel id this user is connected to |
 | user_id | snowflake | the user id this voice state is for |
-| member? | [member object](#DOCS_RESOURCES_GUILD/member-object) | the guild member this voice state is for |
+| member? | [member](#DOCS_RESOURCES_GUILD/member-object) object | the guild member this voice state is for |
```

Comment:
`#DOCS_RESOURCES_GUILD/`**`guild-`**`member-object` -- `member-object` isn't a header on the guild page.